src/client/app/modules/markupControllers/comment.js

```diff
@@ -6,6 +6,30 @@
       this.defineModels(model);
       this.defineMarkup();
       this.setEventsForModalHandling();
+      this.characterLimit = 450;
+      this.$characterLimitCounterSpan.html(this.characterLimit);
+      
+      this.$target.on("change paste keyup", () => {
+        // limit possible text to the character limit, always
+        this.$target.val(this.$target.val().substr(0, this.characterLimit));
+        
+        const newContentString = this.$target.val();
+        const newContentStringLength = newContentString.length;
+        
+        console.log("Counter a " + newContentStringLength);
+        this.$counter.html(newContentStringLength + " de " + 
+          this.characterLimit + " caracteres disponibles.");
+          
+        if(newContentStringLength > (this.characterLimit - 20)) {
```

Comment:
This if-else statement can be also broken into another function called `handleCounterClass()`